content/docs/guides/ha_scale/perf-scale.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Performance and Scalability"
+description: "Performance benchmark on control plane and data plane sidecar and scalability test of Open Service Mesh"
+type: docs
+---
+
+Open Service Mesh aims to create a high-performance service mesh while providing all the essential service mesh features. This document describes the test process and results we performance to evaluate the performance and scalability of OSM.
+
+## Testing environment
+
+This test is performed on a Kubernetes cluster provisioned and managed by [Microsoft Azure Kubernetes Service](https://azure.microsoft.com/en-us/services/kubernetes-service/). The cluster has 100 Standard_DS2_v2 nodes (each node has 2 vCPUs and 7GB memory). It is ensured that there is only one OSM controller pod running with default resource quota during the test. OSM is configured in non-permissive traffic mode.
+
+## Mesh topology
+
+The testing mesh contains two types of service. One is a load generator (modified from [wrk2](https://github.com/giltene/wrk2)), which can send requests at specified rate, load balanced among all other services in the mesh. The load generator serves as the client side of a request. Throughout our test, there will be at most one load generator service. The other type of service is a simple echo application that replies with whatever it receives from an inbound request. It serves as the server side of a request. We can deploy multiple copies of the echo application in the mesh. In short, the mesh topology is a typical 1-to-N service architecture.
+
+<p align="center">
+  <img src="/docs/images/perf/mesh-topo.png" width="650"/>
+</p>
+
+Note that since both types of service are in the mesh and permissive mode is disabled, thus each request will be authenticated via mTLS. It means that requests go through the Envoy sidecar in both the client and the server.
+
+## Metrics
+
+The test result includes a lot of metrics collected. A clear definition of each metric can help avoid ambiguity. Below are the metric definitions we will use throughout this documentation:
+
+* Control plane
+    * Peak time CPU/Memory: the maximum CPU/memory usage during the testing process.
+    * Stable time CPU/Memory usage: the average CPU/memory usage when the control plane is off the peak time and becomes stable.
```

Comment:
same for this to call it average. let's also change the graph labels

Reply:
Providing more context on why, calling it "peak time" is a bit misleading, since it implies that there is a time in which there is a peak, and a time in which there is a max. When in reality, there is not a specific time, and both can happen simultaneously

Reply:
Definition for control plane seems obvious. But some metrics in data plane might have different understanding. E.g. Max usage can be interpreted as the avg of max usage of all sidecars, or the max usage of all sidecars. I suggest we don't completely remove this section.

Reply:
Regarding the "peak time", in our test workflow, we define the mesh scale. The peak time usage is a good indication of the worst case that the mesh will encounter. In the real case, if we slowly adds workload to the mesh and reach a certain scale, there is unlikely to have a spike. But this result does not mean much for evaluating OSM.